2001-3000/LIT2930RepCh211.xml

```diff
@@ -6,8 +6,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <titleStmt>
             <title xml:lang="gez" xml:id="t1">መልክአ፡ አቡነ፡ ተክለ፡ ሃይማኖት፡</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">Malkǝʾa ʾabuna Takla Hāymānot</title>
-            <title xml:lang="en" corresp="#t1">Image of Takla Hāymānot</title>
-            <title xml:lang="en" xml:id="t2">Salām to Takla Hāymānot</title>
+            <title xml:lang="en" corresp="#t1">Malkǝʾ-hymn to Takla Hāymānot</title>
+            <title xml:lang="en" xml:id="t2">Image of Takla Hāymānot</title>
+            <title xml:lang="en" xml:id="t3">Salām to Takla Hāymānot</title>
```

Comment:
I would recomment to avoid the title "Salam to ... " for malke'-hymns. "Salam" is a composition of completely different, to my knowledge, they never overlap in the traditional literature.

Reply:
Good point